app/services/idv/flows/capture_doc_flow.rb

```diff
@@ -8,6 +8,8 @@ class CaptureDocFlow < Flow::BaseFlow
 
       ACTIONS = {
         reset: Idv::Actions::ResetAction,
+        verify_document: Idv::Actions::VerifyDocumentAction,
+        verify_document_status: Idv::Actions::VerifyDocumentStatusAction,
```

Comment:
reason this is needed: these polling actions for async work were missing from the hybrid flow